bin/main.rb

```diff
@@ -1,2 +1,42 @@
 #!/usr/bin/env ruby
-puts 'Helo World!!!!'
+
+# Welcome users and show the board
+
+puts 'Welcome to the tic-tac-toe game'
+
+# Below is where the board will be displayed
+
+puts '| || || |'
+puts '-----------'
+puts '| || || |'
+puts '-----------'
+puts '| || || |'
+puts 'The winner occupies any horizontal, vertical or diagonal positions concurrently'
+
+# Tell players the rules of the game and what is expected of them
+
+puts 'Letters, spaces and special characters are not allowed!! Choose a number from 1 to 9'
+
+# Display the current positions that are taken, at the begining it will be empty.
+puts "Player One it's your turn"
+
+# Get inputs by player 1 and player 2
+
+# player1_input = gets.chomp
+
+# Move of player one displayed on the board
+
+puts 'Player Two its your turn'
+
```

Comment:
##### Implementation details

-  [ ] A loop is needed in this game (recommended is `while` but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png)).
- [ ] Inside the loop, you should call actions repeated for each move.
- [ ] A loop needs to be terminated at some point (after a winning move or draw to be more specific). It can be done with variable Consider the following snippet below for inspiration on the same: 👇 

```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```